tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "strictNullChecks": true,
```

Comment:
I found these yesterday and want to make them visible to the other contributors 😄